Goal: Add "Debian" as vendor suffix

Fixes: nothing

Status wrt upstream: Debian specific

Note: Should be kept

Index: samba-3.0.25b/source/VERSION
===================================================================
--- samba-3.0.25b.orig/source/VERSION	2007-06-27 09:25:31.744419574 +0200
+++ samba-3.0.25b/source/VERSION	2007-06-27 09:28:46.235199543 +0200
@@ -95,5 +95,5 @@
 # e.g. SAMBA_VERSION_VENDOR_SUFFIX=vendor_version()    #
 #  ->  "CVS 3.0.0rc2-VendorVersion"                    #
 ########################################################
-SAMBA_VERSION_VENDOR_SUFFIX=
+SAMBA_VERSION_VENDOR_SUFFIX="Debian"
 SAMBA_VENDOR_PATCH=
